spec/models/admin_create_auction_spec.rb

```diff
@@ -0,0 +1,148 @@
+require 'rails_helper'
+
+RSpec.describe AdminCreateAuction do
+  let(:creator) { AdminCreateAuction.new(params) }
+  let(:auction) { creator.auction }
+
+  describe '#perform' do
+    context 'when the price is too high' do
+      let(:params) {
+        {
+          auction: {
+            title: 'title',
+            description: 'description',
+            github_repo: 'github url',
+            start_datetime: 'Nov 3, 2015',
+            end_datetime: '11/10/2015',
+            start_price: 3500.01
+          }
+        }
+      }
+
+      it 'drops the price down to 3500' do
```

Comment:
I'm a little bit confused by these few specs. The description says the price will be dropped to `3500`, but the assertion (which passes!) says `3400.99`.

1) Is there an incrementor pushing the price down `100.01`?
2) Should the spec descriptions be updated?